R/inline.R

```diff
@@ -16,8 +16,7 @@
 ## along with RcppArmadillo.  If not, see <http://www.gnu.org/licenses/>.
 
 inlineCxxPlugin <- function(...) {
-    ismacos <- Sys.info()[["sysname"]] == "Darwin"
-    openmpflag <- if (ismacos) "" else "$(SHLIB_OPENMP_CFLAGS)"
+    openmpflag <- "$(SHLIB_OPENMP_CFLAGS)"
```

Comment:
Shouldn't this be `SHLIB_OPENMP_CXXFLAGS` instead?

```suggestion
    openmpflag <- "$(SHLIB_OPENMP_CXXFLAGS)"
```